media-fonts/ibm_plex/ibm_plex-0.5.3.recipe

```diff
@@ -0,0 +1,34 @@
+SUMMARY="The package of IBM’s new typeface, IBM Plex"
```

Comment:
`SUMMARY` [shouldn't refer][0] to the package name

[0]: https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#summary

Reply:
I'd make an exception here too, and allow the use of "IBM Plex" because that's not exactly the name of the package, and it also comes at the very end of the description.

Reply:
"The package of" is quite useless, still.

I would use some text from the introduction blog post:

Description:

A font family designed to convey the IBM identity on everything from product packaging to the ibm.com website to development code.

The brand identity conveyed by a distinctive typeface creates a consistent look and feel across a company’s landscape, evident in everything from advertising to corporate blog posts to developer code.

The font family comprises eight line weights, in homage to the eight-bar IBM logo. Each weight is available in serif, sans serif and mono-sans, a monospace font aimed at developers.

Summary:

A font family designed to convey the IBM identity

Reply:
I actually pulled that summary from [here](https://github.com/IBM/type) and the description from [here](https://ibm.github.io/type/).  I really didn't have an opinion of what should be expressed in the SUMMARY or the DESCRIPTION.  What you have above sounds great.

Reply:
Cool thanks!
I was going to update your branch, but noticed that you based your PR on your master branch.
I don't know if it will work, but I'm very curious about it.
And even if I did not try to update your branch you would still have to resync it with the project's master branch. So, let's see what happens :)

(If you need any guidance to resync your master branch I can help you.)